chia/wallet/nft_wallet/nft_wallet.py

```diff
@@ -898,8 +898,13 @@ async def make_nft1_offer(
                         abs(amount),
                         Offer.ph(),
                         primaries=[
-                            AmountWithPuzzlehash({"amount": p.amount, "puzzlehash": Offer.ph(), "memos": []})
-                            for _, p in payments
+                            AmountWithPuzzlehash(
+                                {
+                                    "amount": uint64(sum(p.amount for _, p in payments)),
+                                    "puzzlehash": Offer.ph(),
+                                    "memos": [],
+                                }
+                            )
```

Comment:
Instead of a bunch of payments, one payment that is the sum of all their amounts